pyqtorch/noise/gates.py

```diff
@@ -17,7 +17,7 @@ def __init__(
         self,
         kraus: list[Tensor],
         target: int,
-        error_probabilities: tuple[float, ...] | float,
+        error_probability: tuple[float, ...] | float,
```

Comment:
hm since we allow for several error probability values, I'd still keep the plural.

Reply:
I thought about this, but it is meant to keep everything consistent, since in the noise gates themselves the argument is `error_probability`. Also, while two of them accept several values, most only accept one.